src/the_way/mod.rs

```diff
@@ -229,13 +236,11 @@ impl TheWay {
         Ok(())
     }
 
-    /// Lists snippets (optionally filtered)
-    fn list(&self, filters: &Filters) -> color_eyre::Result<()> {
-        let mut snippets = self.filter_snippets(filters)?;
-        snippets.sort_by(|a, b| a.index.cmp(&b.index));
+    /// Prints given snippets
+    fn show_snippets(&self, snippets: &Vec<Snippet>) -> color_eyre::Result<()> {
```

Comment:
`&Vec<Snippet>` can be replaced by `&[Snippet` here (see [slice](https://doc.rust-lang.org/std/vec/struct.Vec.html#slicing))